build/musicbrainz-dev/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM metabrainz/base-image
+LABEL Maintainer="yvanzo@metabrainz.org"
+
+ENV DOCKERIZE_VERSION v0.5.0
+RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
+    tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
+    rm -f dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+
+ARG POSTGRES_VERSION=9.5
+RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+    echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    apt-key adv --keyserver keyserver.ubuntu.com --refresh-keys 'Yarn' && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+    curl -sLO https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
+    curl -sLO https://deb.nodesource.com/node_10.x/pool/main/n/nodejs/nodejs_10.15.0-1nodesource1_amd64.deb && \
+    apt-get update && \
+    apt-get install \
+        --no-install-recommends \
+        --no-install-suggests \
+        -y \
+        ./google-chrome-stable_current_amd64.deb \
+        ./nodejs_10.15.0-1nodesource1_amd64.deb \
+        cpanminus \
+        bash-completion \
+        build-essential \
+        gettext \
+        git-core \
+        language-pack-de \
+        language-pack-el \
+        language-pack-es \
+        language-pack-et \
+        language-pack-fi \
+        language-pack-fr \
+        language-pack-it \
+        language-pack-ja \
+        language-pack-nl \
+        language-pack-sq \
+        libdb-dev \
+        libexpat1-dev \
+        libicu-dev \
+        liblocal-lib-perl \
+        libpq-dev \
+        libxml2-dev \
+        libssl-dev \
+        memcached \
+        postgresql-${POSTGRES_VERSION} \
+        python-minimal \
+        wget \
+        yarn \
+        zlib1g-dev && \
+    apt-get clean && \
```

Comment:
The Debian and Ubuntu Images from the Docker Library run this automatically.

Reply:
This default behavior should be checked for the current base image which is neither of these.

Reply:
The metabrainz base image is based on [phusion/baseimage:0.9.22](https://github.com/phusion/baseimage-docker/blob/0.9.22/image/Dockerfile) which is based on ubuntu 16.04.
I installed git before in the docker image and now tree and didn't run any other commands in between and it run ``apt-get clean`` automatically.

````
apt install --no-install-recommends git tree
Reading package lists... Done
Building dependency tree       
Reading state information... Done
git is already the newest version (1:2.7.4-0ubuntu1.7).
The following NEW packages will be installed:
  tree
0 upgraded, 1 newly installed, 0 to remove and 104 not upgraded.
Need to get 40.6 kB of archives.
After this operation, 138 kB of additional disk space will be used.
Do you want to continue? [Y/n] 
Get:1 http://archive.ubuntu.com/ubuntu xenial/universe amd64 tree amd64 1.7.0-3 [40.6 kB]
Fetched 40.6 kB in 0s (255 kB/s)
Selecting previously unselected package tree.
(Reading database ... 11590 files and directories currently installed.)
Preparing to unpack .../tree_1.7.0-3_amd64.deb ...
Unpacking tree (1.7.0-3) ...
Setting up tree (1.7.0-3) ...
root@c1fb1a52fd02:/# tree /var/cache/apt/archives/
/var/cache/apt/archives/
├── lock
└── partial

1 directory, 1 file
root@c1fb1a52fd02:/# 
`````

Under my desktop the cache is almost 800MB:

````
sudo diskus /var/cache/apt/archives/
715.87 MB (715,874,304 bytes)
````

So it is proven that ``apt-get clean`` is run automatically.

Reply:
Removed it from while rebasing, see https://github.com/metabrainz/musicbrainz-docker/pull/118/commits/992af4e2db7c30247981372561f4d23358d133b1#diff-bfa80dfe6fc07e10f8447a73a919129cR48.